### PR TITLE
Fix documentation for pendulum_control.

### PR DESCRIPTION
**Pull request by clalancette**
This fixes several minor issues pointed out by rosdoc2 when
building the documentation for pendulum_control.

Signed-off-by: Chris Lalancette <clalancette@openrobotics.org>

### CODE REVIEW

File: pendulum_control/include/pendulum_control/pendulum_controller.hpp [hunk 1]
```diff
@@ -97,6 +97,8 @@ class PendulumController
     message_ready_ = true;
   }
 
+  /// Callback when a pendulum_setpoint topic is received
```
Comment (hidmic) [inline pendulum_control/include/pendulum_control/pendulum_controller.hpp:100]: @clalancette nit: 
```suggestion
  /// Callback for when a pendulum setpoint message is received
```

File: pendulum_control/include/pendulum_control/rtt_executor.hpp [hunk 2]
```diff
@@ -58,6 +58,8 @@ class RttExecutor : public rclcpp::Executor
     return rclcpp::ok() && running;
   }
 
+  /// Fill in the output RttestResults message with data from the executor
```
Comment (hidmic) [inline pendulum_control/include/pendulum_control/rtt_executor.hpp:61]: @clalancette nit: consider explaining in prose what those `RttestResults` are.

**Review by hidmic — APPROVED** [review]
LGTM but for a few nits.

File: pendulum_control/include/pendulum_control/pendulum_controller.hpp (hunk 1 above)
Comment (clalancette) [reply pendulum_control/include/pendulum_control/pendulum_controller.hpp]: Updated in eec9408

File: pendulum_control/include/pendulum_control/rtt_executor.hpp (hunk 2 above)
Comment (clalancette) [reply pendulum_control/include/pendulum_control/rtt_executor.hpp]: Updated in eec9408

**Comment by clalancette** [comment]
Since there is actual code change in this PR (the change from PI to M_PI, in particular), I'm going to run CI on it on all platforms.

CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/15562/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/10246/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/13230/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/15776/)

**Comment by clalancette** [comment]
I'm going to go ahead and merge this one with green CI and an approval, thanks for the review!

**Pull request merged**